src/app/Home.tsx

```diff
@@ -47,6 +51,7 @@ export default function Home({ session }: HomeProps) {
     mint()
```

Comment:
Should use onComple and onError of the useMutation and use the requried mint result to see if it was minted instead of using setMinted

Reply:
I have a query here. If the user has already minted once, should we allow user to mint once again after he refreshes the page or should we show some information like you have already minted the drop.

Reply:
Should use the fact that the mint isn't null after its minted to show the checkmark like before.